python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml

```
@@ -64,6 +66,33 @@ keys:
               convert_types:
                 - int
               default: use_default_mgmt_method_vrf
+      export_to_cloudvision:
+        type: dict
+        description: |-
+          Enables automatic sFlow export to CloudVision.
+          sFlow will be configured if at least one destination is set in `destinations` or if `export_to_cloudvision.enabled: true`.
+          One of them is required.
+        keys:
+          enabled:
+            type: bool
+            description: |-
+              When set to `true`, AVD automatically:
+              - Configures an sFlow destination at `127.0.0.1` port `6343` in the VRF determined by `export_to_cloudvision.vrf`.
+              - Configures the TerminAttr daemon (`daemon terminattr`) to export sFlow to the same `export_to_cloudvision.vrf/127.0.0.1:6343` in the selected VRF.
```

@ClausHolbechArista for help on phrasing:

```suggestion
              Configures an sFlow destination for `127.0.0.1` port `6343` in the VRF defined in `export_to_cloudvision.vrf`.
              Also configures the TerminAttr daemon to export sFlow to this destination.
```